dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -229,3 +229,43 @@ LOGINPROPERTY_OVERRIDE_USERFILTER.useraction=If the userFilter attribute is pref
 INVALID_LOGIN_PROPERTIES=CWIML4506E: The following login properties are not valid: {0}.
 INVALID_LOGIN_PROPERTIES.explanation=One or more login properties are not valid WIM PersonAccount properties.
 INVALID_LOGIN_PROPERTIES.useraction=Either choose valid PersonAccount properties or add the properties to PersonAccount as extended properties. Ensure that the case for each login property matches the case for the corresponding PersonAccount property.
+
+KRB5_LOGIN_FAILED_CACHE=CWIML4507E: Kerberos login failed using Kerberos principal {0} and Kerberos credential cache (ccache) {1}.
+KRB5_LOGIN_FAILED_CACHE.explanation=Either the specified Kerberos principal is invalid, or the Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_CACHE.useraction=Ensure that a valid Kerberos principal is specified, and that the Kerberos credential cache (ccache) is not expired.
+
+KRB5_LOGIN_FAILED_KEYTAB=CWIML4508E: Kerberos login failed using Kerberos principal {0} and Kerberos keytab {1}.
+KRB5_LOGIN_FAILED_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_KEYTAB.useraction=Ensure that a valid Kerberos principal is specified, and that a valid Kerberos keytab containing the Kerberos principal is specified.
+		
+KRB5_LOGIN_FAILED_DEFAULT_CACHE=CWIML4509E: Kerberos login failed using Kerberos principal {0} and the default Kerberos credential cache (ccache).
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.useraction=Ensure that a valid Kerberos principal is specified, and that the default Kerberos credential cache (ccache) is not expired.
+
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB=CWIML4510E: Kerberos login failed using Kerberos principal {0} and the default Kerberos keytab.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.useraction=Ensure that a valid Kerberos principal is specified and that the default Kerberos keytab is valid.
+
+KRB5_TICKETCACHE_USED=CWIML4511E: The {0} LdapRegistry is configured with a Kerberos ticket cache (ccache) filename {1} and keytab filename {2}. The Kerberos credential cache (ccache) is used for Kerberos bind authentication to LDAP server.
```

Comment:
The {0} LdapRegistry is configured with the {1} Kerberos ticket cache (ccache) filename and the {2} keytab filename.